tests/test_org_sessions_api.py

```diff
@@ -0,0 +1,120 @@
+import itertools
```

Comment:
I know I told you to emulate the existing sessions API tests, but it occurs to me that this entity should only ever be used by SnQL. So you can test it with SnQL only. See `test_snql_sdk_api.py` for an example of what fixtures you would need and what the tests look like. I don't think you would need to change the `setup_method` here but you don't need `test_entity`, `test_app` etc. Those are all for porting legacy tests to SnQL.

Reply:
Converted this to SnQL only. Changed the post method and removed those two fixtures.